Api/Fluent Validation.md

```diff
@@ -74,14 +74,7 @@ public class LoginDetailsDtoValidator : AbstractValidator<LoginDetailsDto>
 
     public bool ValidateUsername(string username)
     {
-         if(doesNameAlreadyExistInDatabase)
-         {
-             return false;
-         }
-         else
-         {
-             return true;
-         }
+        return doesUsernameAlreadyExistInDatabase ? false : true
```

Comment:
While we're optimizing this code - the `doesUsernameAlreadyExistInDatabase` field comes out of nowhere here, and also if we assume that the field is boolean, we are returning a boolean value based on a single value. Could you please improve this example? It's perfectly fine if we have some empty methods in our examples because the point is to show the `FluentValidation` functionalities rather than how to implement specific validation rules like checking whether a username is already in the database.